components/aws/sagemaker/tests/integration_tests/scripts/run_integration_tests

```diff
@@ -229,10 +233,16 @@ else
   wait
 fi
 
-generate_oidc_role_name
-"$cwd"/generate_iam_role ${EKS_CLUSTER_NAME} ${OIDC_ROLE_NAME} ${REGION} ${KFP_NAMESPACE} ${KFP_SERVICE_ACCOUNT}
+if [[ "${SKIP_OIDC_SETUP}" == "false" ]]; then
+  generate_oidc_role_name
+  "$cwd"/generate_iam_role ${EKS_CLUSTER_NAME} ${OIDC_ROLE_NAME} ${REGION} ${KFP_NAMESPACE} ${KFP_SERVICE_ACCOUNT}
+fi
+
 install_kfp
-install_oidc_role
+
+if [[ "${SKIP_OIDC_SETUP}" == "false" ]]; then
```

Comment:
long pending fix, thanks Kartik

can we change this to skip_odic_kfp_setup ? I think thats gonna be more useful ?